egs/yomdle_tamil/v1/local/chain/run_cnn_e2eali_1b.sh

```diff
@@ -0,0 +1 @@
+tuning/run_cnn_e2eali_1b.sh
```

Comment:
The scripts which are not in the tuning directory do not need to have a suffix (like 1b). They usually point to the best recipe in the tuning directory. Please rename this to `run_cnn_e2eali.sh`